Answers.md

```diff
@@ -1,11 +1,16 @@
 # Answers
 
 1.  What is React JS and what problems does it try and solve?
+    It is a JS library that uses components and other backend functionality to create a virtual DOM that makes it possible for websites to be more performant. It was created by facebook to solve the problem of too much data and on each browser change. 

```

Comment:
It's not technically backend stuff. We're still dealing purely with front end tech.